core/src/martian/yaml.clj

```diff
@@ -0,0 +1,32 @@
+(ns martian.yaml
```

Comment:
I wouldn't mind a test for this ns, just at a very basic level (single route)

Reply:
Okay